ProcessMaker/Http/Controllers/Api/ProcessController.php

```diff
@@ -902,7 +902,7 @@ public function preimportValidation(Process $process, Request $request)
 
         if (!$result = $this->validateImportedFile($content, $request)) {
             return response(
-                ['message' => __('The selected file is invalid or not supported for import.')],
+                ['message' => __('The selected file is is a invalid or not supported for the Process importer. Please verify that this file is a Process.')],
```

Comment:
Can we restore the original error message 'The selected file is invalid or not supported for import.'

Reply:
Closing, saw that you updated the text for process templates too.